tests/timeseries/mock_resampler.py

```diff
@@ -61,7 +61,7 @@ def power_senders(
                 self._input_channels_receivers[name] = [
                     self._channel_registry.get_or_create(
                         Sample[Quantity], name
-                    ).new_receiver()
+                    ).new_receiver(limit=1)
```

Comment:
This doesn't feel right, because we want the raw data to queue up if we're not consuming continuously.  What was going wrong without this change?

Reply:
It was needed before( When we had no synchronization between MetricFetcher and FallbackMetricFetcher).

Reply:
This is still pending.